hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/codec/DecodingFacade.java

```diff
@@ -38,765 +35,988 @@
+    private static final ABIType<Tuple> ASSOCIATE_TOKEN_DECODER =
+            TypeFactory.create(ADDRESS_PAIR_RAW_TYPE);
+
+    private static final Function DISSOCIATE_TOKENS_FUNCTION =
+            new Function("dissociateTokens(address,address[])", INT_OUTPUT);
+    private static final Bytes DISSOCIATE_TOKENS_SELECTOR =
+            Bytes.wrap(DISSOCIATE_TOKENS_FUNCTION.selector());
+    private static final ABIType<Tuple> DISSOCIATE_TOKENS_DECODER =
+            TypeFactory.create("(bytes32,bytes32[])");
+
+    private static final Function DISSOCIATE_TOKEN_FUNCTION =
+            new Function("dissociateToken(address,address)", INT_OUTPUT);
+    private static final Bytes DISSOCIATE_TOKEN_SELECTOR =
+            Bytes.wrap(DISSOCIATE_TOKEN_FUNCTION.selector());
+    private static final ABIType<Tuple> DISSOCIATE_TOKEN_DECODER =
+            TypeFactory.create(ADDRESS_PAIR_RAW_TYPE);
+
+    private static final Function TOKEN_URI_NFT_FUNCTION =
+            new Function("tokenURI(uint256)", STRING_OUTPUT);
+    private static final Bytes TOKEN_URI_NFT_SELECTOR =
+            Bytes.wrap(TOKEN_URI_NFT_FUNCTION.selector());
+    private static final ABIType<Tuple> TOKEN_URI_NFT_DECODER =
+            TypeFactory.create(UINT256_RAW_TYPE);
+
+    private static final Function BALANCE_OF_TOKEN_FUNCTION =
+            new Function("balanceOf(address)", INT_OUTPUT);
+    private static final Bytes BALANCE_OF_TOKEN_SELECTOR =
+            Bytes.wrap(BALANCE_OF_TOKEN_FUNCTION.selector());
+    private static final ABIType<Tuple> BALANCE_OF_TOKEN_DECODER = TypeFactory.create("(bytes32)");
+
+    private static final Function OWNER_OF_NFT_FUNCTION =
+            new Function("ownerOf(uint256)", INT_OUTPUT);
+    private static final Bytes OWNER_OF_NFT_SELECTOR = Bytes.wrap(OWNER_OF_NFT_FUNCTION.selector());
+    private static final ABIType<Tuple> OWNER_OF_NFT_DECODER = TypeFactory.create(UINT256_RAW_TYPE);
+
+    private static final Function ERC_TRANSFER_FUNCTION =
+            new Function("transfer(address,uint256)", BOOL_OUTPUT);
+    private static final Bytes ERC_TRANSFER_SELECTOR = Bytes.wrap(ERC_TRANSFER_FUNCTION.selector());
+    private static final ABIType<Tuple> ERC_TRANSFER_DECODER =
+            TypeFactory.create("(bytes32,uint256)");
+
+    private static final Function WIPE_TOKEN_ACCOUNT_FUNCTION =
+            new Function("wipeTokenAccount(address,address,uint32)", INT_OUTPUT);
+
+    private static final Bytes WIPE_TOKEN_ACCOUNT_SELECTOR =
+            Bytes.wrap(WIPE_TOKEN_ACCOUNT_FUNCTION.selector());
+
+    private static final ABIType<Tuple> WIPE_TOKEN_ACCOUNT_DECODER =
+            TypeFactory.create("(bytes32,bytes32,uint32)");
+
+    private static final Function WIPE_TOKEN_ACCOUNT_NFT_FUNCTION =
+            new Function("wipeTokenAccountNFT(address,address,int64[])", INT_OUTPUT);
+
+    private static final Bytes WIPE_TOKEN_ACCOUNT_NFT_SELECTOR =
+            Bytes.wrap(WIPE_TOKEN_ACCOUNT_NFT_FUNCTION.selector());
+
```

Comment:
nit: remove white line, so that WIPE_TOKEN_NFT variables are structured together